multitenancy-spring-boot-autoconfigure/src/main/java/org/axonframework/extensions/multitenancy/autoconfig/MultiTenancyAxonServerAutoConfiguration.java

```diff
@@ -108,17 +108,21 @@ public TenantCommandSegmentFactory tenantAxonServerCommandSegmentFactory(
             AxonServerConfiguration axonServerConfig,
             AxonServerConnectionManager connectionManager
     ) {
-        return tenantDescriptor -> AxonServerCommandBus.builder()
-                                                       .localSegment(localSegment)
-                                                       .serializer(messageSerializer)
-                                                       .routingStrategy(routingStrategy)
-                                                       .priorityCalculator(priorityCalculator)
-                                                       .loadFactorProvider(loadFactorProvider)
-                                                       .targetContextResolver(targetContextResolver)
-                                                       .axonServerConnectionManager(connectionManager)
-                                                       .configuration(axonServerConfig)
-                                                       .defaultContext(tenantDescriptor.tenantId())
-                                                       .build();
+        return tenantDescriptor -> {
```

Comment:
With this changes, there's no difference between the `registerTenant` and `registerTenantAndStart` methods anymore, right?
Both methods typically invoke their respective tenant segment factories, but started is no longer post poned with this adjustment.